packages/cli/src/utils/api.js

```diff
@@ -96,6 +97,7 @@ const callAPI = async (
   }
 
   debug(`>> ${requestOptions.method} ${requestOptions.url || res.url}`);
+  debug(`>> Request Headers: ${JSON.stringify(requestOptions.headers)}`);
```

Comment:
Would it print sensitive data like the deploy key to the console? If so, we should avoid it.